sockeye/arguments.py

```diff
@@ -758,6 +758,10 @@ def add_training_args(params):
                               default=(.0, .0),
                               help='Recurrent dropout without memory loss (Semeniuta, 2016) for encoder & decoder '
                                    'LSTMs. Use "x:x" to specify separate values. Default: %(default)s.')
+    train_params.add_argument('--rnn-enc-last-hidden-concat-to-embedding',
+                              action="store_true",
+                              help='Concatenate the last hidden of the encoder to the input of the decoder, '
+                                   'instead of the previous state of the decoder.')
```

Comment:
You could add a "Default: %(default)s." at the end of the help message.

Reply:
Ok